tests/hardhat/Fork/constants.ts

```diff
@@ -0,0 +1,90 @@
+import GovernanceBscMainnet from "@venusprotocol/governance-contracts/deployments/bscmainnet.json";
+import GovernanceBscTestnet from "@venusprotocol/governance-contracts/deployments/bsctestnet.json";
+
+// TESTNET DEPLOYED CONTRACTS
+import { contracts as MainnetContracts } from "../../../deployments/bscmainnet.json";
+// MAINNET DEPLOYED CONTRACTS
+import { contracts as TestnetContracts } from "../../../deployments/bsctestnet.json";
+
+export const contractAddreseses = {
+  sepolia: {
+    ADMIN: "0x02EB950C215D12d723b44a18CfF098C6E166C531",
+    ACM: "0x92587A8799CaC6C0A5184218c050B3E19790ac0B",
```

Comment:
Ideally we can use `await (ethers.getContract('NAME_OF_CONTRACT)).address` to get address. This will make this more resilient in the event the order of the deployments changes the tests won't break because of a changed address

Reply:
nvm: Actually since this is a forked test they won't change

Reply:
I was not using the deployed addresses for sepolia as there are some contracts which are not deployed yet on Sepolia, once they are we can.
Additionally there are some addresses which are not present in the deployment files like:-
- TOKEN1, TOKEN2
- ACC1, ACC2, and ...others.

So, we will need to maintain different objects for separate networks and using both `await (ethers.getContract('NAME_OF_CONTRACT)).address` and objects for networks can compromise readability I think.
Wdyt?